core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -431,9 +434,11 @@ class KryoSerializerSuite extends SparkFunSuite with SharedSparkContext {
     ser.deserialize[HashMap[Int, List[String]]](serializedMap)
   }
 
-  private def testSerializerInstanceReuse(autoReset: Boolean, referenceTracking: Boolean): Unit = {
+  private def testSerializerInstanceReuse(
+     autoReset: Boolean, referenceTracking: Boolean, usePool: Boolean): Unit = {
```

Comment:
Total nit: there are a couple continuation indents that are 3 rather than 4 spaces. Don't bother, unless you need to make other updates.